docs/content/doc/installation/on-cloud-provider.md

```diff
@@ -35,10 +35,16 @@ you can experiment with running Gitea.
 
 Gitea can found in [Vultr](https://www.vultr.com)'s marketplace.
 
-To deploy it have a look at https://www.vultr.com/marketplace/apps/gitea.
+To deploy Gitea to Vultr, have a look at [the Vultr Marketplace listing](https://www.vultr.com/marketplace/apps/gitea).
```

Comment:
Personally, I would put the `the` outside of the link in all three cases…

Reply:
```suggestion
To deploy Gitea to Vultr, have a look at the [Vultr Marketplace](https://www.vultr.com/marketplace/apps/gitea).
```